### ISSUE TITLE
Joint-genotyping across samples

### ISSUE THREAD

**Issue by jjfarrell** [eblerjana/pangenie#18]
Is there a way to generate a joint-genotyped vcf with pangenie?   If I understand the software correctly, only the variants in the pangenome reference are called across all the genotyped samples while the  non-pangenome variants detected  are called within individual samples.

**Comment by eblerjana** [comment]
No, unfortunately there is no joint genotyping mode. PanGenie genotypes a single sample in each run. Also, PanGenie is a re-genotyping method, so it only genotypes the variants contained in the pangenome input VCF. It does not detect any new ones.
